packages/components/multiselect/src/Multiselect.tsx

```diff
@@ -331,47 +293,60 @@ function _Multiselect(props: MultiselectProps, ref: React.Ref<HTMLDivElement>) {
                       : 'Clear selection'
                   }
                   size="small"
+                  withTooltip
+                  tooltipProps={{
+                    content: clearButtonProps.tooltip
+                      ? clearButtonProps.tooltip
+                      : 'Clear selection',
+
+                    showDelay: 800,
+                    placement: 'top',
+                    as: 'div',
+                  }}
                 />
-              </Tooltip>
-            </div>
-          )}
-        </Flex>
-        <Popover.Content
-          ref={mergeRefs(listRef, internalListRef)}
-          className={cx(
-            styles.content(listMaxHeight),
-            popoverProps.className,
-            styles.container,
-          )}
-          testId="cf-multiselect-container"
-          onBlur={() => onBlur?.()}
-        >
-          <FocusLock focusOptions={{ preventScroll: true }} returnFocus={true}>
-            {hasSearch && (
-              <MultiselectSearch
-                {...searchProps}
-                setSearchValue={setSearchValue}
-                searchValue={searchValue}
-                focusList={focusList}
-              />
+              </div>
             )}
-            {isLoading && <ListItemLoadingState />}
-
-            {!isLoading && optionsLength > 0 && (
-              <ul className={styles.list} data-test-id="cf-multiselect-items">
-                {hasSearch ? enrichOptions(children) : children}
```

Comment:
Why are we removing this logic? Is it implemented somewhere else or are we changing the behavior?